weave/weave_server.py

```diff
@@ -379,16 +380,47 @@ def frontend(path):
 @blueprint.route("/", defaults={"path": None})
 @blueprint.route("/<path:path>")
 def root_frontend(path):
+    if request.args.get("unsetBetaVersion") is not None:
```

Comment:
should this behavior be made available to all installations? e.g. server and saas/prod? I can see it is useful for qa or server qa installs, but unclear if we want to expose that for prod users as well.

Reply:
The way it is right now, it would be enabled for everyone. Ideally, it would be enabled just for cloud beta, but unless I'm mistaken we don't have a cloud beta environment at the moment. I don't think there's any way for this feature to be discovered by normal users, so I feel pretty safe enabling it even on weave.wandb.ai.

It probably won't work properly on server, but this being undiscoverable also applies there. We could explicitly disable it for server if that's necessary.